src/main.jsx

```diff
@@ -94,8 +95,10 @@ ReactDOM.createRoot(document.getElementById("root")).render(
       <AuthContextProvider>
         <JobContextProvider>
           <JobSaveContextProvider>
-            <RouterProvider router={router} />
-            <TailWindToaster />
+            <UserContextProvider>
```

Comment:
Actually, for most times, this context is not need except we are accessing the admin page. try searching for `AdminContextProvider`, that's where I think it could be better to be placed at?

Reply:
ya thats a good point

Reply:
moved it into admin